### PR TITLE
Add getMotd() and setMotd() methods to the Server interface

### PR DESCRIPTION
**Pull request by ausbin**
Adds `getMotd()` and `setMotd()` methods to the Server interface and the Bukkit class. 

This coupled with [my pull request in CraftBukkit](https://github.com/Bukkit/CraftBukkit/pull/794) will allow plugins to change the motd with one function call and optionally save it to the config file.

Ticket: https://bukkit.atlassian.net/browse/BUKKIT-1799

### CODE REVIEW

**Comment by gravitylow** [comment]
Needs a ticket

**Comment by ausbin** [comment]
Sorry, here's the ticket: https://bukkit.atlassian.net/browse/BUKKIT-1799

**Comment by turt2live** [comment]
Already kinda possible: https://github.com/Bukkit/Bukkit/blob/master/src/main/java/org/bukkit/event/server/ServerListPingEvent.java#L47

**Comment by ausbin** [comment]
@turt2live thanks for pointing that out! I didn't know that existed. 

To be honest, I really don't know whether or not to close this request.

Although it's probably better for the plugin to have its own config file and use that event, it might be handy in some cases to run one api call and save the motd to server.properties instead of intercepting each server list ping.

**Comment by turt2live** [comment]
Leave it open.

I was just pointing out it's possible, but as you mentioned, not optimally possible.

Let the Bukkit team decide I guess

File: src/main/java/org/bukkit/Server.java [hunk 1]
```diff
@@ -639,4 +654,4 @@
      * @returns The water animal spawn limit
      */
     int getWaterAnimalSpawnLimit();
-}
\ No newline at end of file
+}
```
Comment (jrtc27) [inline src/main/java/org/bukkit/Server.java:657]: What's up with this?

Comment (turt2live) [inline src/main/java/org/bukkit/Server.java:657]: Means a new line was added

Comment (jrtc27) [inline src/main/java/org/bukkit/Server.java:657]: And it shouldn't be there, should it?

Comment (ausbin) [inline src/main/java/org/bukkit/Server.java:657]: For some reason `Server.java` didn't (and still doesn't) have a newline at the end, but I looked at some of the other Java files in the repository and they do. Vim just added a newline at the end when I edited it and diff is reflecting that. More info [here](http://stackoverflow.com/questions/5813311/no-newline-at-end-of-file).

Comment (turt2live) [inline src/main/java/org/bukkit/Server.java:657]: It's not a breaking change, just a \n at the end jrtc27.

It can be ignored.

**Comment by emericask8ur** [comment]
I like the idea.. Even if it has been implemented in an event.

**Comment by feildmaster** [comment]
see pr in craftbukkit.

**Pull request closed without merging**